### PR TITLE
Update unsorted.m3u

### PR DESCRIPTION
**Pull request by freearhey**
Removed broken links.

### CODE REVIEW

File: channels/unsorted.m3u [hunk 1]
```diff
@@ -75,24 +47,14 @@ https://5b57bb229a2e6.streamlock.net/live/televizija/playlist.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Bitlis TV (720p) [Not 24/7]
 https://59cba4d34b678.streamlock.net/canlitv/bitlistv/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",BR Fernsehen
-https://mcdn.br.de/bfs_sued/de/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Canal 26
 http://live-edge01.telecentro.net.ar:1935/live/26hd-720/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Canal 29
 https://59f1cbe63db89.streamlock.net:1443/canal/_definst_/canal/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Canal Acequia (720p) [Not 24/7]
-https://api.new.livestream.com/accounts/6450028/events/5813077/live.m3u8
```
Comment (jkemp814) [inline channels/unsorted.m3u:93]: Currently, streaming. 10/29/21 15:42

File: channels/unsorted.m3u [hunk 2]
```diff
@@ -235,8 +169,6 @@ http://51.15.154.138/providus/live2805_hq/index.m3u8
 https://w2.kringvarp.fo/uttanlands/_definst_/smil:uttanlands.smil/chunklist_w587901821_b772000_slfao.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",La Muscle TV
 https://streamcdn.lamuscle.com/lamtv-origin/smil:monsterworkout-tricepspt1.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Latest TV
-https://5a0e89631aa14.streamlock.net/LatestTV/LatestTV/playlist.m3u8
```
Comment (jkemp814) [inline channels/unsorted.m3u:239]: Currently, streaming 10/29/21 15:47

File: channels/unsorted.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/unsorted.m3u:93]: Streams here too.

File: channels/unsorted.m3u [hunk 3]
```diff
@@ -435,30 +341,18 @@ https://vod1.laki.eu/sandzak/video.m3u8
...
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Shota TV
-https://t2.tvmak.com/tv/SHOTATV.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",SimSi TV
 https://simultv.s.llnwi.net/o060/SimSiTV/interlink.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Slap Tech
 https://simultv.s.llnwi.net/o061/SlapTech/interlink.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="http://190.211.140.13/soltv/img/logomenu.png" group-title="",Sol Televisión
 http://190.211.140.89:8081/SVTranscoder/SOLTVabr.stream/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Sonus FM TV
-https://60015180a7f57.streamlock.net:444/public/stream_source/chunklist_w1017805699.m3u8
```
Comment (jkemp814) [inline channels/unsorted.m3u:449]: Currently, streaming. 10/29/21 15:52

File: channels/unsorted.m3u [hunk 4]
```diff
@@ -105,26 +67,18 @@ https://5d8d85cf2c308.streamlock.net:1936/CanalcostaTV/WXP6YT/playlist.m3u8
...
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Comfy TV
-https://simultv.s.llnwi.net/o054/ComfyTV/interlink.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Contact Vision TV
 http://contactvision.flashmediacast.com:1935/contactvision/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CostadelSol TV
 https://limited11.todostreaming.es/live/benalmadena-livestream.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Cowboy Theater
 https://simultv.s.llnwi.net/o054/CowboyTheater/interlink.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CROTV HD
-http://92.204.40.139/crotv/televizijahrvatskedijaspore/index.m3u8
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:127]: Changed to http://92.204.40.139:8081/crotv/televizijahrvatskedijaspore/playlist.m3u8

File: channels/unsorted.m3u [hunk 5]
```diff
@@ -163,38 +115,26 @@ https://d12a2vxqkkh1bo.cloudfront.net/hls/1080p/playlist.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",First
 http://us.bestvideostreaming.is/8717/index.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Flutura TV
-https://t2.tvmak.com/tv/fluturatv.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Folk TV
 http://584b0aa350b92.streamlock.net:1935/folk-tv/myStream.sdp/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Friesischer Rundfunk Friedeburg
 https://583548fa0d82b.streamlock.net/friesischerRundfunk-live/_definst_/mp4:friesischerrundfunk/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Fuel TV
-https://stream.ads.ottera.tv/playlist.m3u8?network_id=571
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:173]: They might be relying on Pluto TV now that they got it.

File: channels/unsorted.m3u [hunk 6]
```diff
@@ -541,26 +419,12 @@ http://us.bestvideostreaming.is/8716/playlist.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",TV S1
 https://sradio.ipradio.rs/sradio/radiostv/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",TV Sharri
 http://uk4.streamingpulse.com:1935/TVSHARRI/TVSHARRI/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",TV Tema
 http://134.209.238.38/hls/test.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",TV Tropical
-https://59f2354c05961.streamlock.net:1443/tvtropical/_definst_/tvtropical/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",TV União (720p) [Not 24/7]
-https://596639ebdd89b.streamlock.net/tvuniao/tvuniao/playlist.m3u8
```
Comment (jkemp814) [inline channels/unsorted.m3u:559]: Currently, streaming. 10/29/21 15:56

**Review by jkemp814 — DISMISSED** [review]

**Review by Sphinxroot — CHANGES_REQUESTED** [review]
all https://t2.tvmak.com/tv/   are now  https://t1.tvmak.com/tv/

https://tvmak.com/

**Comment by LaneSh4d0w** [comment]
> 
> 
> all https://t2.tvmak.com/tv/ are now https://t1.tvmak.com/tv/
> 
> https://tvmak.com/

Referer locked too.

File: channels/unsorted.m3u [hunk 7]
```diff
@@ -75,24 +47,16 @@ https://5b57bb229a2e6.streamlock.net/live/televizija/playlist.m3u8
 http://us.bestvideostreaming.is/8795/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Belarus 24
 http://serv30.vintera.tv:8081/belarus24/belarus24/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Best TV
-https://s.best-tv.gr/HLS/hls/best/1_2/index.m3u8
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:79]: Moved to https://stream.net7.gr/hls/best/index.m3u8

File: channels/unsorted.m3u [hunk 8]
```diff
@@ -75,24 +47,16 @@ https://5b57bb229a2e6.streamlock.net/live/televizija/playlist.m3u8
...
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Beteve
 https://cdnapisec.kaltura.com/p/2346171/sp/234617100/playManifest/entryId/1_n6442jz0/format/applehttp/protocol/https/uiConfId/42816492/a.m3u8?referrer=aHR0cHM6Ly9iZXRldmUuY2F0
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Beykent TV
-https://beykenttv.arsivnet.net/P62039948/beykenttv/chunklist_w1942204286.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Bitlis TV (720p) [Not 24/7]
 https://59cba4d34b678.streamlock.net/canlitv/bitlistv/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",BR Fernsehen
-https://mcdn.br.de/bfs_sued/de/master.m3u8
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:87]: Moved to https://brcdn.vo.llnwd.net/br/fs/bfs_sued/hls/de/master.m3u8

File: channels/unsorted.m3u [hunk 9]
```diff
@@ -163,38 +119,26 @@ https://d12a2vxqkkh1bo.cloudfront.net/hls/1080p/playlist.m3u8
...
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Gurbani Healing
-http://live.wmncdn.net/gurbanihealing/e09c9e2a70cc03a0b6d2171391dd682b.sdp/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Haldensleben TV
 https://578d8e1867e87.streamlock.net/medienasa-vod/_definst_/mp4:hdl_sendung_720p.mp4/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",HD 365
 https://netstreaming.eu:8080/hls/hd365.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Hi Music
-https://t2.tvmak.com/tv/himusic.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Hipodromo de las Americas
-http://wms30.tecnoxia.com/soelvi/abr_soelvi/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:189]: From deduction, changed to http://wms10.tecnoxia.com/soelvi/slv423/playlist.m3u8

File: channels/unsorted.m3u [hunk 10]
```diff
@@ -105,26 +69,20 @@ https://5d8d85cf2c308.streamlock.net:1936/CanalcostaTV/WXP6YT/playlist.m3u8
...
-https://simultv.s.llnwi.net/o054/ComfyTV/interlink.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Contact Vision TV
 http://contactvision.flashmediacast.com:1935/contactvision/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CostadelSol TV
 https://limited11.todostreaming.es/live/benalmadena-livestream.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Cowboy Theater
 https://simultv.s.llnwi.net/o054/CowboyTheater/interlink.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",CROTV HD
-http://92.204.40.139/crotv/televizijahrvatskedijaspore/index.m3u8
+http://92.204.40.139:8081/crotv/televizijahrvatskedijaspore/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:85]: CROTV -> Czech Republic

**Comment by Sphinxroot** [comment]
> > all https://t2.tvmak.com/tv/ are now https://t1.tvmak.com/tv/
> > https://tvmak.com/
> 
> Referer locked too.

i tryed with referer  And  user agent  nothing worked for me seem to be some kind of script hiding https://tvmak.com/assets/hls.js

because when i download a m3u8  as exemple : https://t1.tvmak.com/tv/TVFESTA.m3u8?vDC2NncsIf
it download this https://tvmak.com/tvmak.m3u8  insted

**Comment by LaneSh4d0w** [comment]
> 
> 
> > > all https://t2.tvmak.com/tv/ are now https://t1.tvmak.com/tv/
> > > https://tvmak.com/
> > 
> > 
> > Referer locked too.
> 
> i tryed with referer And user agent nothing worked for me seem to be some kind of script hiding https://tvmak.com/assets/hls.js
> 
> because when i download a m3u8 as exemple : https://t1.tvmak.com/tv/TVFESTA.m3u8?vDC2NncsIf it download this https://tvmak.com/tvmak.m3u8 insted
> 
> if i streamlink it to kodi it work plugin://plugin.video.streamlink-tester/?action=play&url=https://tvmak.com/TVFESTA

Autocloses on my side with referer https://tvmak.com/

File: channels/unsorted.m3u (hunk 1 above)
Comment (eltoncomp) [reply channels/unsorted.m3u:93]: Canal 26 is from Argentina

File: channels/unsorted.m3u (hunk 6 above)
Comment (eltoncomp) [reply channels/unsorted.m3u:557]: TV União is from Fortaleza and Natal, Brazil

File: channels/unsorted.m3u (hunk 9 above)
Comment (freearhey) [reply channels/unsorted.m3u:189]: This link doesn't work for me either.

File: channels/unsorted.m3u (hunk 10 above)
Comment (freearhey) [reply channels/unsorted.m3u:85]: Moved the link to `de.m3u`

According to [these](https://www.lyngsat.com/tvchannels/de/CroTV-HD.html) the channel is broadcast from Germany.

File: channels/unsorted.m3u (hunk 9 above)
Comment (LaneSh4d0w) [reply channels/unsorted.m3u:189]: Might be Not 24/7.

File: channels/unsorted.m3u [hunk 11]
```diff
@@ -1,64 +1,46 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="Arabic" tvg-logo="http://www.daawah.tv/img/logo.png" group-title="Religious",Daawah TV
 http://51.15.246.58:8081/daawahtv/daawahtv2/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",1A Network
-https://simultv.s.llnwi.net/n4s4/2ANetwork/interlink.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",21 Mix
-https://t2.tvmak.com/tv/RTV21MIX.m3u8
+#EXTVLCOPT:http-referrer=https://tvmak.com/
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:93.0) Gecko/20100101 Firefox/93.0
+https://t1.tvmak.com/tv/RTV21MIX.m3u8?n7rKivxDfY
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:7]: Should work now. 21 Mix -> Albania

File: channels/unsorted.m3u [hunk 12]
```diff
@@ -1,64 +1,46 @@
...
-https://t2.tvmak.com/tv/AANEWYORK.m3u8
+#EXTVLCOPT:http-referrer=https://tvmak.com/
+#EXTVLCOPT:http-user-agent=Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:93.0) Gecko/20100101 Firefox/93.0
+https://t1.tvmak.com/tv/AANEWYORK.m3u8?n7rKivxDfY
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Adria Music RS
 http://91.212.150.248/AdriaMusicTV/index.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",AdriaNet
 http://79.106.48.2:4578/live/adriamed/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Ahi TV Kirsehir
 http://yayin3.canlitv.com:1935/canlitv/ahitv/chunklist.m3u8
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:21]:
```suggestion
http://yayin3.canlitv.com:1935/canlitv/ahitv/playlist.m3u8
```

Ahi TV -> Kirsehir, Turkey

File: channels/unsorted.m3u [hunk 13]
```diff
@@ -75,24 +57,16 @@ https://5b57bb229a2e6.streamlock.net/live/televizija/playlist.m3u8
...
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Beykent TV
-https://beykenttv.arsivnet.net/P62039948/beykenttv/chunklist_w1942204286.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Bitlis TV (720p) [Not 24/7]
 https://59cba4d34b678.streamlock.net/canlitv/bitlistv/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",BR Fernsehen
-https://mcdn.br.de/bfs_sued/de/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Canal 26
 http://live-edge01.telecentro.net.ar:1935/live/26hd-720/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Canal 29
 https://59f1cbe63db89.streamlock.net:1443/canal/_definst_/canal/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:67]:
```suggestion
https://59f1cbe63db89.streamlock.net:1443/canal//canal/playlist.m3u8
```

Most certainly from Brazil

File: channels/unsorted.m3u [hunk 14]
```diff
@@ -139,8 +105,6 @@ https://ghaasiflu.online/tarab/tracks-v1a1/mono.m3u8
 https://ghaasiflu.online/Dijlah/tracks-v1a1/mono.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Dimensions
 https://simultv.s.llnwi.net/o054/Dimensions/interlink.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Domaci TV
-https://stream.domaci.com.hr:3865/live/domacitvlive.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",dorf tv
 https://stream.openplayout.org/hls/dorf_high/index.m3u8
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:109]:
```suggestion
https://stream.openplayout.org/hls/dorftv/live.m3u8
```

Germany

File: channels/unsorted.m3u [hunk 15]
```diff
@@ -297,8 +237,6 @@ http://tv.balkanweb.com:8081/news24/livestream/playlist.m3u8
 https://news18lokmat-lh.akamaihd.net/i/n18lokmat_1@178974/index_5_av-p.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Nisville TV RS
 http://92.60.237.32:1935/live/nisville/playlist.m3u8
-#EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",Noovo
-https://bcsecurelivehls-i.akamaihd.net/hls/live/551061/618566855001/master.m3u8
 #EXTINF:-1 tvg-id="" tvg-country="" tvg-language="" tvg-logo="" group-title="",OF-TV Offenbach
 https://5864df9ceac85.streamlock.net/germanpictures-live/_definst_/mp4:streamschedule/playlist.m3u8
```
Comment (LaneSh4d0w) [inline channels/unsorted.m3u:241]:
```suggestion
https://5864df9ceac85.streamlock.net/germanpictures-live/mp4:streamschedule/playlist.m3u8
```
https://www.of-tv.de/ Germany

**Comment by farman1294** [comment]
Please file 🙏indian to Pakistan

**Pull request closed without merging**